flink-queryable-state/flink-queryable-state-client-java/src/main/java/org/apache/flink/queryablestate/network/ChunkedByteBuf.java

```diff
@@ -88,6 +102,16 @@ public ByteBuf readChunk(ChannelHandlerContext ctx) throws Exception {
 		}
 	}
 
+	@Override
+	public long length() {
+		return length;
```

Comment:
Is it always guaranteed that `buf.readableBytes` won't change? If not, then I think we should return `-1` here, because the length is unknown.

Reply:
It seems like content of `buf` is not changing. However we can not guard against it programmatically, only via java doc. On the other hand, I'm not sure what would the side effects of "unknown" length be.

Reply:
changed to `-1`